geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/CreateIndexFunction.java

```diff
@@ -93,6 +91,31 @@ public void execute(FunctionContext context) {
     }
   }
 
+  private void setResultInSender(FunctionContext context, IndexInfo indexInfo, String memberId,
+      Cache cache, String regionPath) {
+    if (regionPath == null) {
+      String message = CliStrings.format(CliStrings.CREATE_INDEX__INVALID__REGIONPATH,
+          indexInfo.getRegionPath());
+      context.getResultSender().lastResult(new CliFunctionResult(memberId, false, message));
+    } else {
+      XmlEntity xmlEntity =
+          new XmlEntity(CacheXml.REGION, "name", cache.getRegion(regionPath).getName());
+      context.getResultSender().lastResult(new CliFunctionResult(memberId, xmlEntity));
+    }
+  }
+
+  private String getValidRegionName(Cache cache, String regionPath) {
+    while (cache.getRegion(regionPath) == null && regionPath != null) {
```

Comment:
If regionPath is null, will cache.getRegion(regionPath) throw an exception?  Shouldn't the regionPath check for null be first?